Validation: Description-missing fixes. (#234)
MicrosoftDocs/bingmaps-docs: BingMaps/v8-web-control/modules/spatial-data-service-module/getboundaryrequestoptions-object.md

---
title: "GetBoundaryRequestOptions Object | Microsoft Docs"

> description: Describes the GetBoundaryRequestOptions object, which contains the options for GeoData API requests boundary data, and provides its properties.

Reviewer: ```suggestion
description: Describes the GetBoundaryRequestOptions object, which contains the options for GeoData API requests boundary data, with descriptions for each of its properties.
```
